src/python/WMComponent/DBS3Buffer/DBSBufferBlock.py

```diff
@@ -455,14 +456,14 @@ def FillFromDBSBuffer(self, blockInfo):
         self.startTime = blockInfo.get('creation_date')
         self.inBuff    = True
 
-        if 'status' in blockInfo.keys():
+        if 'status' in blockInfo.keys():  # did not modernize, lack of test
             self.status = blockInfo['status']
             if self.status == "Pending":
                 self.data['block']['open_for_writing'] = 0
 
             del blockInfo['status']
 
-        for key in blockInfo.keys():
+        for key in blockInfo.keys():  # did not modernize, lack of test
```

Comment:
```suggestion
        for key in blockInfo:
```
should be enough, I guess.